src/js/player.js

```diff
@@ -2763,8 +2773,14 @@ class Player extends Component {
    * and calls `reset` on the tech`.
    */
   reset() {
+    if (this.tech_) {
+      this.tech_.clearTracks(['text']);
+    }
     this.loadTech_(this.options_.techOrder[0], null);
     this.techCall_('reset');
+    if (isEvented(this)) {
```

Comment:
I don't think this check is necessary because the Player will always has event handling.

Reply:
Player starts html5 tech and then resets test fails without it. It does not make a lot of sense though